custom/icds/tasks.py

```diff
@@ -23,9 +24,16 @@
 AWC_LOCATION_TYPE_CODE = 'awc'
 SUPERVISOR_LOCATION_TYPE_CODE = 'supervisor'
 
+ANDHRA_PRADESH_SITE_CODE = '28'
+MAHARASHTRA_SITE_CODE = ''
```

Comment:
we don't have this yet so for now it's ignored by being left blank. we'll just fill it in once we have it